### PR TITLE
My COVID-Extensions

### PR DESCRIPTION
**Pull request by JuanMartin10**
Here is my COVID Extensions for Chrome.
In this extensions when you push '+' button, you can show the lastest global covid data 
Thanks :)

Here's my video:
https://vimeo.com/405561259
Captura de pantalla 2020-04-08 a las 21 29 52

### CODE REVIEW

**Comment by amandena** [comment]
Hey Juan, the link to the video didn't load on my end -- not sure if I need to have a Vimeo account to view?

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+p{
+  font-size: 12px;
+  margin: 5px 0  5px 20px
+}
+
+.country {
+  max-width: 50%;
+  padding: 5px;
+  margin: 0 0 20px 0
```
Comment (npkorintus) [inline src/index.css:29]: can this just be `margin-bottom: 20px` here?

File: src/components/Covid.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                </div>
+
+                <h2>Cases By Country</h2>
+
+
+                <div className="inline">
+                    {data.Countries.map(elm => (
+                        <div className='country'>
+                            <h5 key={elm.Country}>{elm.Country}</h5>
+                            <hr></hr>
```
Comment (npkorintus) [inline src/components/Covid.js:52]: what are these `<hr>` tags for?

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "short_name": "News Extension",
+  "name": "Breaking News Extension",
+  "browser_action": {
+    "default_popup": "index.html",
+    "default_icon": "hello_extensions.png"
```
Comment (amandena) [inline public/manifest.json:6]: If you add a default_title it will display your text, i.e. "Breaking News Extension"

**Comment by lexholder** [comment]
I have a vimeo account and the link isn't working for me either

File: src/components/Covid.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                    <p>Total confirmed: {data.Global.TotalConfirmed}</p>
+                    <p>New confirmed: {data.Global.NewConfirmed}</p>
+                    <p>New deaths: {data.Global.NewDeaths}</p>
+                    <p>New recovered: {data.Global.NewRecovered}</p>
+                    <p>Total recovered: {data.Global.TotalRecovered}</p>
+                </div>
+
+                <h2>Cases By Country</h2>
+
+
```
Comment (amandena) [inline src/components/Covid.js:47]: There are a lot of extra spaces that can be removed for readability

**Comment by JuanMartin10** [comment]
I'm sorry guys, im a noob too in Vimeo.. I think its already done

File: src/components/Covid.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+const Covid = () => {
+
+    const [data, setData] = useState(null);
+
+    useEffect(() => {
+        axios.get('https://api.covid19api.com/summary')
+            .then(response => setData(response.data))
+            .catch(error => console.log(error))
```
Comment (carloshdelreal) [inline src/components/Covid.js:12]: try to use the async await sintax is way more readable

File: src/components/Covid.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+            if (a.TotalConfirmed > b.TotalConfirmed) {
+                return -1;
+            }
+            if (a.TotalConfirmed < b.TotalConfirmed) {
+                return 1;
+            }
+            return 0;
+
+        })
+
```
Comment (carloshdelreal) [inline src/components/Covid.js:27]: It is not clear what is this logic there, if is doing something to the data so it should be inside of the useEffect function

File: src/index.css (hunk 1 above)
Comment (JuanMartin10) [reply src/index.css:29]: Yes! I forgot change it, because in the begining I code other margin and I foget to change it! haha

**Pull request closed without merging**